### ISSUE TITLE
Blank image on fedora linux flatpak LXDE

### ISSUE THREAD

**Issue by iza001** [imikado/dupotEasyFlatpak#20]
For approximately 4 updates the app loads, but does not show anything, if I try to search for something, the search results appear

![image](https://github.com/user-attachments/assets/cfe1d74a-205a-4d4e-8971-40e0040ff304)

When doing a search, it appears like this: 

![image](https://github.com/user-attachments/assets/e0a32ae7-da72-4ead-92d7-95fa4ad3d054)

```
flatpak run org.dupot.easyflatpak 
(dupot_easy_flatpak:2): Gtk-WARNING **: 21:28:09.730: Theme parsing error: gtk.css:3297:43: Expected ')' in color definition
flutter: app directory already there
flutter: build installed is the latest (3.1.5)
flutter: start application
flutter: init application
flutter: statePage: loading
flutter: Installation
flutter: End installation
flutter:  flathub ok
flutter: check updates
flutter: Null check operator used on a null value
flutter: #0      LocalizationApi.tr (package:dupot_easy_flatpak/Infrastructure/Api/localization_api.dart:48)
flutter: #1      _SideMenuViewState.build (package:dupot_easy_flatpak/Infrastructure/Screen/View/side_menu_view.dart:151)
flutter: #2      StatefulElement.build (package:flutter/src/widgets/framework.dart:5729)
flutter: #3      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5617)
flutter: #4      StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5780)
flutter: #5      Element.rebuild (package:flutter/src/widgets/framework.dart:5333)
flutter: #6      ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5599)
flutter: #7      StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5771)
flutter: #8      ComponentElement.mount (package:flutter/src/widgets/framework.dart:5593)
flutter: #9      Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #10     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #11     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6900)
flutter: #12     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #13     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #14     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5642)
flutter: #15     Element.rebuild (package:flutter/src/widgets/framework.dart:5333)
flutter: #16     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5599)
flutter: #17     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5593)
flutter: #18     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #19     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #20     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5642)
flutter: #21     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5780)
flutter: #22     Element.rebuild (package:flutter/src/widgets/framework.dart:5333)
flutter: #23     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5599)
flutter: #24     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5771)
flutter: #25     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5593)
flutter: #26     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #27     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #28     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6900)
flutter: #29     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #30     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #31     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5642)
flutter: #32     Element.rebuild (package:flutter/src/widgets/framework.dart:5333)
flutter: #33     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5599)
flutter: #34     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5593)
flutter: #35     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #36     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #37     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6900)
flutter: #38     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #39     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #40     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5642)
flutter: #41     Element.rebuild (package:flutter/src/widgets/framework.dart:5333)
flutter: #42     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5599)
flutter: #43     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5593)
flutter: #44     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #45     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #46     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6900)
flutter: #47     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #48     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #49     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5642)
flutter: #50     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5780)
flutter: #51     Element.rebuild (package:flutter/src/widgets/framework.dart:5333)
flutter: #52     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5599)
flutter: #53     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5771)
flutter: #54     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5593)
flutter: #55     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #56     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #57     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5642)
flutter: #58     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5780)
flutter: #59     Element.rebuild (package:flutter/src/widgets/framework.dart:5333)
flutter: #60     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5599)
flutter: #61     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5771)
flutter: #62     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5593)
flutter: #63     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #64     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #65     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6900)
flutter: #66     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #67     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #68     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6900)
flutter: #69     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #70     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #71     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5642)
flutter: #72     Element.rebuild (package:flutter/src/widgets/framework.dart:5333)
flutter: #73     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5599)
flutter: #74     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5593)
flutter: #75     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #76     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #77     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6900)
flutter: #78     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #79     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #80     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6900)
flutter: #81     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #82     MultiChildRenderObjectElement.inflateWidget (package:flutter/src/widgets/framework.dart:7035)
flutter: #83     MultiChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:7047)
flutter: #84     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #85     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #86     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5642)
flutter: #87     Element.rebuild (package:flutter/src/widgets/framework.dart:5333)
flutter: #88     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5599)
flutter: #89     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5593)
flutter: #90     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #91     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #92     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5642)
flutter: #93     Element.rebuild (package:flutter/src/widgets/framework.dart:5333)
flutter: #94     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5599)
flutter: #95     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5593)
flutter: #96     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4468)
flutter: #97     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #98     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5642)
flutter: #99     Element.rebuild (package:flutter/src/widgets/framework.dart:5333)
flutter: check updates
flutter: remove Instance of 'MaterialPage<dynamic>'
flutter: Another exception was thrown: Instance of 'DiagnosticsProperty<void>'
flutter: Another exception was thrown: Instance of 'DiagnosticsProperty<void>'
flutter: Another exception was thrown: Instance of 'DiagnosticsProperty<void>'
```

**Comment by imikado** [comment]
hi,
can you test in darkmode ?

because of the interface issue, you can edit directly the json file

It is here: Documents/EasyFlatpak/userSettings.json
you can enbable darkMode by modify boolean value here:
replace
`"darkModeEnabled": false,`
by
`"darkModeEnabled": true,`
then launch the application again

ps: does it work before ? 

You are on fedora, LXDE, waylan or x11 ?

**Comment by iza001** [comment]
> hi, can you test in darkmode ?
> 
> because of the interface issue, you can edit directly the json file
> 
> It is here: Documents/EasyFlatpak/userSettings.json you can enbable darkMode by modify boolean value here: replace `"darkModeEnabled": false,` by `"darkModeEnabled": true,` then launch the application again
> 
> ps: does it work before ?
> 
> You are on fedora, LXDE, waylan or x11 ?

Hi again!
I use fedora on X11, the problem persists! and I am with the new version 3.1.7. I have tried flatpak, and dnf, but the same thing appears:
![image](https://github.com/user-attachments/assets/7ebf8cda-bca5-4075-8964-6d4562905685)

Jason file: 
```{
    "version": 1,
    "applicationDataPath": "",
    "userOverrideLanguageCode": false,
    "languageCode": "en",
    "userOverrideDarkModeEnabled": true,
    "darkModeEnabled": true,
    "userInstallationScopeEnabled": false,
    "displayApplicationInstalledNumberInSideMenu": false,
    "displayApplicationInstalledNumberInPage": false
}
```

**Comment by imikado** [comment]
Hi,
i m looking for the error, i don't find the reason, can you test with this json ?
```
{   
     "version": 1,
      "applicationDataPath": "",
      "userOverrideLanguageCode": true,
      "languageCode": "en",
      "userOverrideDarkModeEnabled": true,
      "darkModeEnabled": true,
      "userInstallationScopeEnabled": false,
      "displayApplicationInstalledNumberInSideMenu": false,
      "displayApplicationInstalledNumberInPage": false
}
```

and can you launch it in terminal to have more error oinformation with this json file ?
`flatpak run org.dupot.easyflatpak`

**Comment by iza001** [comment]
here it is!

```
flatpak run org.dupot.easyflatpak 
Gtk-Message: 00:50:47.541: Failed to load module "xapp-gtk3-module"

(dupot_easy_flatpak:2): Gtk-WARNING **: 00:50:47.645: Theme parsing error: gtk.css:3297:43: Expected ')' in color definition
flutter: app directory already there
flutter: build installed is the latest (3.1.7)
flutter: start application
flutter: Installation
flutter: End installation
flutter:  flathub ok
flutter: check updates
flutter: Null check operator used on a null value
flutter: #0      LocalizationApi.tr (package:dupot_easy_flatpak/Infrastructure/Api/localization_api.dart:48)
flutter: #1      _SideMenuViewState.build (package:dupot_easy_flatpak/Infrastructure/Screen/View/side_menu_view.dart:149)
flutter: #2      StatefulElement.build (package:flutter/src/widgets/framework.dart:5743)
flutter: #3      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5631)
flutter: #4      StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5794)
flutter: #5      Element.rebuild (package:flutter/src/widgets/framework.dart:5347)
flutter: #6      ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5613)
flutter: #7      StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5785)
flutter: #8      ComponentElement.mount (package:flutter/src/widgets/framework.dart:5607)
flutter: #9      Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #10     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #11     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6914)
flutter: #12     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #13     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #14     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5656)
flutter: #15     Element.rebuild (package:flutter/src/widgets/framework.dart:5347)
flutter: #16     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5613)
flutter: #17     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5607)
flutter: #18     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #19     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #20     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5656)
flutter: #21     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5794)
flutter: #22     Element.rebuild (package:flutter/src/widgets/framework.dart:5347)
flutter: #23     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5613)
flutter: #24     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5785)
flutter: #25     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5607)
flutter: #26     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #27     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #28     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6914)
flutter: #29     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #30     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #31     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5656)
flutter: #32     Element.rebuild (package:flutter/src/widgets/framework.dart:5347)
flutter: #33     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5613)
flutter: #34     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5607)
flutter: #35     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #36     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #37     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6914)
flutter: #38     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #39     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #40     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5656)
flutter: #41     Element.rebuild (package:flutter/src/widgets/framework.dart:5347)
flutter: #42     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5613)
flutter: #43     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5607)
flutter: #44     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #45     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #46     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6914)
flutter: #47     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #48     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #49     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5656)
flutter: #50     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5794)
flutter: #51     Element.rebuild (package:flutter/src/widgets/framework.dart:5347)
flutter: #52     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5613)
flutter: #53     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5785)
flutter: #54     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5607)
flutter: #55     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #56     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #57     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5656)
flutter: #58     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5794)
flutter: #59     Element.rebuild (package:flutter/src/widgets/framework.dart:5347)
flutter: #60     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5613)
flutter: #61     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5785)
flutter: #62     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5607)
flutter: #63     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #64     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #65     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6914)
flutter: #66     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #67     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #68     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6914)
flutter: #69     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #70     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #71     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5656)
flutter: #72     Element.rebuild (package:flutter/src/widgets/framework.dart:5347)
flutter: #73     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5613)
flutter: #74     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5607)
flutter: #75     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #76     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #77     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6914)
flutter: #78     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #79     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #80     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6914)
flutter: #81     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #82     MultiChildRenderObjectElement.inflateWidget (package:flutter/src/widgets/framework.dart:7049)
flutter: #83     MultiChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:7061)
flutter: #84     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #85     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #86     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5656)
flutter: #87     Element.rebuild (package:flutter/src/widgets/framework.dart:5347)
flutter: #88     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5613)
flutter: #89     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5607)
flutter: #90     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #91     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #92     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5656)
flutter: #93     Element.rebuild (package:flutter/src/widgets/framework.dart:5347)
flutter: #94     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5613)
flutter: #95     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5607)
flutter: #96     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4480)
flutter: #97     Element.updateChild (package:flutter/src/widgets/framework.dart:3963)
flutter: #98     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5656)
flutter: #99     Element.rebuild (package:flutter/src/widgets/framework.dart:5347)
flutter: check updates
flutter: Another exception was thrown: Instance of 'DiagnosticsProperty<void>'
flutter: Another exception was thrown: Instance of 'DiagnosticsProperty<void>'
flutter: Another exception was thrown: Instance of 'DiagnosticsProperty<void>'


```

**Comment by imikado** [comment]
Hi,
can you test this version ?
https://github.com/imikado/dupotEasyFlatpak/releases/tag/3.1.8

**Comment by iza001** [comment]
it worked!
I used this version https://github.com/imikado/dupotEasyFlatpak/releases/tag/3.1.8 as a dnf package and it worked! thank you so much!. Solved!

**Comment by imikado** [comment]
for information i justed published the 3.1.9 which fix an error on search page
i just ask flathub to publish this 3.1.9

https://github.com/imikado/dupotEasyFlatpak/releases/tag/3.1.9
